resources/views/queue-size.blade.php

```diff
@@ -2,13 +2,12 @@
     <x-pulse::card-header name="Queue Sizes"
         x-bind:title="`Time: {{ number_format($time) }}ms; Run at: ${formatDate('{{ $runAt }}')};`"
         details="past {{ $this->periodForHumans() }}">
-
         <x-slot:icon>
             <x-pulse::icons.queue-list />
         </x-slot:icon>
     </x-pulse::card-header>
 
-    <div class="flex flex-wrap gap-4">
+    <div class="flex flex-wrap gap-4" style="margin-bottom: 15px;">
```

Comment:
Prefer to put this into the card-header so you don't have to add a manual margin. Anyway, with Tailwind you would add the `mb-3` class instead of adding a manual style.